ckan/tests/logic/auth/test_get.py

```diff
@@ -621,3 +621,11 @@ def test_sysadmin_can_list_followees(self, func):
         sysadmin = factories.Sysadmin()
         context = {"user": sysadmin["name"], "model": model}
         assert helpers.call_auth(func, context=context)
+
+
+@pytest.mark.usefixtures("non_clean_db")
+class TestStatusShow:
+    
+    def test_status_show_is_visible_to_anonymous(self):
+        context = {"user": "", "model": model}
```

Comment:
Can there also be a test with no `user` object in the context at all?

Reply:
I can add it if necessary. However, if I'm not mistaken, there isn't a single test in `auth/test_get.py` with no `user` attribute in the context object.

Reply:
It sounds like it would be a good thing to have at least once. I'm pretty sure it won't be a problem, since anything in `get.py` automatically gets the effect of the `auth_allow_anonymous_access` annotation, but that should be tested.

Reply:
It turns out that `call_auth()` has this: 

https://github.com/ckan/ckan/blob/802ec31ccc0d7122fab73a6d84f5d2739fc6d0f8/ckan/tests/helpers.py#L159-L161

So a test with no `user` in `context` won't work.

Reply:
Most tests either use `"user": None` or `"user": ""`, it really doesn't matter which is used as there is no clear convention.